cssdb.json

```
@@ -373,6 +373,15 @@
       }
     ]
   },
+  {
+    "id": "in-out-of-range-pseudo-class",
+    "title": "`:in-range` and `:out-of-range` Pseudo-Classes",
+    "description": "A pseudo-class for matching elements that have range limitations,
```

Looks like we’re missing a `"` and this should be ready to go.

```suggestion
    "description": "A pseudo-class for matching elements that have range limitations",
```